documentation/project-proposal.md

```diff
@@ -131,49 +131,37 @@ Given I am a logged-in user,\
 When I select to remove an existing budget,\
 Then the budget should be successfully removed and not show on the budget tab.
 
-4. As a user, I want to set budgets for specific categories like food and travel so that I can allocate funds to different areas of my life.\
-**Acceptance Criteria:**\
-Given I am a logged-in user,\
-When I set a budget for a specific category,\
-Then I should see the correct budget set for each category.
-
-5. As a user, I want to see how much of my budget I have used so that I can adjust my spending habits.\
+4. As a user, I want to see how much of my budget I have used so that I can adjust my spending habits.\
 **Acceptance Criteria:**\
 Given I am a logged-in user,\
 When I navigate to the budget tab,\
 Then I should see already existing budget and their progress.
 
 ### Expense Analytics
 
-1. As a user, I want visual insights into my expenses so that I can quickly understand my spending habits.\
+1. As a user, I want visual insights into my expenses from the first day of the month to the current day of the month so that I can quickly understand my spending habits.\
```

Comment:
```suggestion
1. As a user, I want visual insights into my daily expenses from the first day of the month to the current day of the month so that I can quickly understand my spending habits.\
```